config_parser.cc

```diff
@@ -150,6 +150,7 @@ NginxConfigParser::TokenType NginxConfigParser::ParseToken(std::istream* input,
 bool NginxConfigParser::Parse(std::istream* config_file, NginxConfig* config) {
   std::stack<NginxConfig*> config_stack;
   config_stack.push(config);
+  printf("%d", int(config_stack.size()));
```

Comment:
Should remove print statements like this from the commit, as they don't have helper information